src/main/resources/wa-task-completion-civil-civil.dmn

```diff
@@ -59,7 +59,7 @@
       <rule id="DecisionRule_1fr79q4">
         <description>Event that is completed by the user in ExUI. Values can be listed as comma separated</description>
         <inputEntry id="UnaryTests_1kohzcy">
-          <text>"CLAIMANT_RESPONSE", "CLAIMANT_RESPONSE_SPEC"</text>
+          <text>"CLAIMANT_RESPONSE", "CLAIMANT_RESPONSE_SPEC", "CREATE_SDO"</text>
```

Comment:
Sorry @sampankumar - I don't really know WA, so this might be a stupid question.
So we are using TAKE_CASE_OFFLINE instead of CASE_PROCEEDS_IN_CASEMAN, and this is just to create a task for a caseworker to take that action?

In that case, does the stateflow not need updating with a TAKE_CASE_OFFLINE event?

Reply:
TAKE_CASE_OFFLINE is the event/task that is mentioned in DMN. This matches what DJ have done to take the case offline as a task